sherlock/resources/data.json

```diff
@@ -214,6 +214,13 @@
     "urlMain": "https://blip.fm/",
     "username_claimed": "blue"
   },
+  "Blipfoto": {
+    "errorMsg": "This user doesn't exist",
+    "errorType": "message",
```

Comment:
Looks like you also changed the `errorType` for Blipfoto. Only LibraryThing had to be changed.
Revert the Blipfoto error checking to `status_code` and this PR should be ready to be merged :)